core/services/aws/tests/user/cognitoTrigger/lambdas/postConfirmationTrigger.test.ts

```diff
@@ -0,0 +1,76 @@
+import postConfirmationLambda from '../../../../user/cognitoTrigger/lambdas/postConfirmationTrigger'
+import { PostConfirmationTriggerEvent } from 'aws-lambda'
+import { UserService } from '@application/userWorkflows/UserService'
+import DynamoDbTableOperations from '../../../../commons/ddb/DynamoDbTableOperations'
+import { UserDTO } from '@commons/dto/UserDTO'
+
+jest.mock('@application/userWorkflows/UserService')
+jest.mock('../../../../commons/ddb/DynamoDbTableOperations')
+
+describe('postConfirmationLambda Tests', () => {
+  let mockEvent: PostConfirmationTriggerEvent
```

Comment:
do we really need ITM style code?
Why can't we shallow copy?

Reply:
put `mockEvent` to a separate file and use shallow copy where needed.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/cdf187f52a5e2767732bf7c75295e46c200058db)